threedsecure/client_test.go

```diff
@@ -32,22 +33,32 @@ func TestThreeDSecureNew(t *testing.T) {
 		ReturnURL: "https://test.com",
 	}
 
-	target, err := New(threeDSecureParams)
+	tdsrc, err := New(threeDSecureParams)
```

Comment:
Sorry for the nitpick-y feedback, but would you mind changing this naming to something a little more descriptive? IMO, abbreviated local names are suitable for very well-scoped use, but it's better to avoid them more generally. (I'm not sure what `tdsrc` is here for example, I'd guess "3-D Secure ResourCe?)

Something like `threeDSecure` would be fine.